src/System.Management.Automation/CoreCLR/EventResource.cs

```diff
@@ -0,0 +1,616 @@
+#if UNIX
+/*
+    This code was generated by the tools\ResxGen\ResxGen.ps1 run against PowerShell-Core-Instrumentation.man.
+    To add or change logged events and the associated resources, edit PowerShell-Core-Instrumentation.man
+    then rerun ResxGen.ps1 to produce an updated CS and Resx file.
+*/
+using System.Collections.Generic;
+using System.Management.Automation.Internal;
+using System.Runtime.InteropServices;
+
+namespace System.Management.Automation.Tracing
+{
+    /// <summary>
+    /// Provides a class for describing a message resource for an ETW event.
+    /// </summary>
+    internal static class EventResource
+    {
+        // Defines the resource id of the message to use when an event id is not valid.
+        const string MissingEventIdResourceName = "MissingEventIdMessage";
+
+        /// <summary>
+        /// Gets the name of the message resource to use for event ids that are not found.
+        /// is not found.
+        /// </summary>
+        /// <remarks>
+        /// This method is called when GetMessage returns a null value indicating the passed
+        /// in event id was not found. The message should be used as the format string
+        /// with the event id as the single variable argument.
+        /// <remarks>
+        public static string GetMissingEventMessage(out int parameterCount)
+        {
+            parameterCount = 1;
+            return MissingEventIdResourceName;
+        }
+
+        /// <summary>
+        /// Gets the message resource id for the specified event id
```

Comment:
Please add final dot.

Reply:
Actually, EventResource is internal.  I'll correct the comments though.